src/twig.core.js

```diff
@@ -913,15 +916,20 @@ var Twig = (function (Twig) {
      *
      */
     Twig.Templates.loadRemote = function(location, params, callback, error_callback) {
-        var id          = params.id,
-            method      = params.method,
-            async       = params.async,
-            precompiled = params.precompiled,
-            template    = null;
+        var id                  = params.id,
+            method              = params.method,
+            async               = params.async,
+            precompiled         = params.precompiled,
+            fetchTemplateSource = params.fetchTemplateSource,
```

Comment:
I like that idea.  That will make it easier to add different types of formats in the future